resolvers_settings_sample.yaml

```diff
@@ -14,11 +14,51 @@ polygon:
     gasLess: true
     rhsSettings:
       mode: None
-      contractAddress: 0x16A1ae4c460C0a42f0a87e69c526c61599B28BC9
+      contractAddress: 0x3d3763eC0a50CE1AdF83d0b5D99FBE0e3fEB43fb
```

Comment:
On all networks rhs should be 0x7dF7...

Reply:
Also I don't remember if we have new rhs url for offchain.

Reply:
I think we dont have a new domain for rhs.